src/vmm/src/snapshot/mod.rs

```diff
@@ -40,6 +41,9 @@ pub use crate::snapshot::persist::Persist;
 #[cfg(target_arch = "x86_64")]
 const SNAPSHOT_MAGIC_ID: u64 = 0x0710_1984_8664_0000u64;
 
+/// Constant bounding how much memory bincode may allocate during vmstate file deserialization
```

Comment:
nit: `//` instead of `///` since it's not a public const

Reply:
I prefer using `///` even for private constants because my IDE will still display the doc comment in a tooltip (and it will show up in `cargo doc --document-private-items`)